package.json

```diff
@@ -1,23 +1,26 @@
 {
   "private": true,
+  "workspaces": [
+    "packages/*"
+  ],
   "scripts": {
-    "build": "node packages/react-scripts/scripts/build.js",
+    "build": "cd packages/react-scripts && node scripts/build.js",
     "changelog": "lerna-changelog",
     "create-react-app": "node tasks/cra.js",
     "e2e": "tasks/e2e-simple.sh",
     "e2e:docker": "tasks/local-test.sh",
-    "postinstall": "node bootstrap.js && cd packages/react-error-overlay/ && npm run build:prod",
-    "publish": "tasks/release.sh",
-    "start": "node packages/react-scripts/scripts/start.js",
+    "postinstall": "cd packages/react-error-overlay/ && yarn build:prod",
+    "publish": "lerna publish --independent",
```

Comment:
I'd really like to see the `git status --porcelain` check back in here somehow.

Reply:
Any ideas? I want to lose that file

Reply:
```bash
(git status --porcelain | grep -q ^ && echo "Your git status is not clean. Aborting." && exit 1 || exit 0) && lerna publish --independent
```